kornia/geometry/transform/pyramid.py

```diff
@@ -350,3 +350,49 @@ def build_pyramid(
         pyramid.append(img_down)
 
     return pyramid
+
+
+def build_laplacian_pyramid(
+    input: torch.Tensor, max_level: int, border_type: str = 'reflect', align_corners: bool = False
```

Comment:
```suggestion
    input: Tensor, max_level: int, border_type: str = 'reflect', align_corners: bool = False
```
`from korna.core import Tensor`

Reply:
same for rest